test/behavior/tendrl/cluster.feature

```diff
@@ -1 +1,91 @@
-Feature: CLUSTER
\ No newline at end of file
+Feature: CLUSTER
+
```

Comment:
the first scenario should be the user is on cluster list page and user is able to see one cluster at least.